src/EnergyPlus/SolarCollectors.cc

```diff
@@ -987,26 +977,24 @@ namespace SolarCollectors {
             // transmittance-absorptance product normal incident:
             Theta = 0.0;
             this->CalcTransRefAbsOfCover(state, Theta, TransSys, RefSys, AbsCover1, AbsCover2);
-            this->TauAlphaNormal = TransSys * state.dataSolarCollectors->Parameters(ParamNum).AbsorOfAbsPlate /
```

Comment:
There were a number of member variables which were simply never used in the rest of the codebase, so they are cleaned out.